python/Ganga/GPIDev/Adapters/IPostProcessor.py

```diff
@@ -83,10 +82,11 @@ def __str__(self):
         else:
             return str(GPIProxyObjectFactory(self.process_objects))
 
-    def append(self, value):
+    def append(self, _value):
+        from Ganga.GPIDev.Proxy import stripProxy
```

Comment:
Indeed, thanks for testing, as it stands I see other problems with this PR which may be show stopper. We also need a proper PR looking into the use of post-processors as they're unfortunately very fragile and have been brought up at least 3 times during 6.1.17 development/bugfixing